src/i18n/locale/en.json

```diff
@@ -36,8 +36,8 @@
           },
           "label": {
             "action": "Add bookmark",
-            "error_one": "Error bookmarking chapter",
-            "error_other": "Error bookmarking chapters",
+            "error_one": "Could not bookmark the chapter",
```

Comment:
chapter is a proper noun

Reply:
Not that it even follows its own advice, but
https://m1.material.io/style/writing.html#writing-language
https://m1.material.io/patterns/errors.html#errors-user-input-errors
"Failed to" is more technical and advanced.
It hearkens to errors one would expect out of a compiler, rather than informative tidbits in general UI.